src/read_config.c

```diff
@@ -18,7 +18,13 @@ along with mfaktc (mfakto).  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include <stdio.h>
+#ifdef _WIN32
 #include <string.h>
+#define strcasecmp _stricmp
+#else // assuming POSIX or BSD compliant system
+#include <strings.h>
```

Comment:
`strings.h` should be avoided until absolutely necessary. I can comment this out on Linux and this file still compiles.

Reply:
My understanding for this was that it was necessary for case insensitive string comparisons on non windows systems. Perhaps I was operating under bad assumptions though.

Reply:
`strings.h` is not a replacement for `string.h`, at least not on Linux with glibc. For instance, including `strings.h` without `string.h` makes `strlen`, `strcpy` and other functions undeclared and break compilation.

`strcasecmp` was indeed introduced in `strings.h`, but glibc makes it available in `string.h`. I can confirm that using `string.h` instead of `strings.h` would not break compilation on MacOS if `strcasecmp` is used (`string.h` includes `strings.h`). I would not worry too much about other non-glibc systems.